internal/pathutil/pathutil.go

```diff
@@ -0,0 +1,41 @@
+// Package pathutil provides utilities for finding test-relative paths at runtime.
+package pathutil
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"sync"
+)
+
+var (
+	mu       sync.Mutex
+	rootPath string
+)
+
+// RootPath returns an absolute path to the root "featureprofiles" directory.
+func RootPath() (string, error) {
+	mu.Lock()
```

Comment:
Is RootPath() expected to be called by multiple go routines?

Reply:
I don't have any such anticipated use cases, but since it was cached I figured "why not" ... I can remove if you prefer

Reply:
Its should be fine.